sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/InsertAdaptiveSparkPlan.scala

```diff
@@ -101,7 +102,6 @@ case class InsertAdaptiveSparkPlan(
     // TODO migrate dynamic-partition-pruning onto adaptive execution.
```

Comment:
So DPP is supported and this comment looks out-of-dated?

Reply:
Done.